app/javascript/pages/getAssistance/housing-counselors.tsx

```diff
@@ -1,13 +1,16 @@
 import { t } from "@bloom-housing/ui-components"
 import React from "react"
-import Layout from "../../layouts/Layout"
+import AssistanceLayout from "../../layouts/AssistanceLayout"
 import withAppSetup from "../../layouts/withAppSetup"
-
+// Counselors can help you with your DAHLIA application and housing search.
 const HousingCounselors = () => {
   return (
-    <Layout title={t("pageTitle.housingCounselors")}>
-      {<h1>{t("pageTitle.housingCounselors")}</h1>}
-    </Layout>
+    <AssistanceLayout
+      title={t("pageTitle.housingCounselors")}
+      subtitle={t("pageSubtitle.housingCounselors")}
```

Comment:
i noticed in the mocks, all of the h1's have only the first word capitalized. Not sure how much this matters 
![image](https://user-images.githubusercontent.com/98352600/222863277-d1899058-9c10-4bad-8a5a-e12e6075e063.png)

Reply:
Yes, Sentence case please.

Reply:
Good catch. Really happy to have your eyes on this too!